packages/edit-navigation/src/components/inspector-additions/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import AutoAddPagesPanel from './auto-add-pages-panel';
+import DeleteMenuPanel from './delete-menu-panel';
+
+export default function InspectorAdditions( { menuId, onDeleteMenu } ) {
+	const selectedBlock = useSelect( ( select ) =>
+		select( 'core/block-editor' ).getSelectedBlock()
```

Comment:
Shouldn't we want to include `[]` as the second argument of `useSelect`?

Reply:
d02bcb8c86